Source/Router.swift

```diff
@@ -101,6 +102,7 @@ public final class Router {
     }
     
     private var routes: [String : Route] = [:]
+    private var canceledRequests: [NSURLProtocol] = []
```

Comment:
Careful, this would create a retain cycle between Server and Router objects. Not sure if we need a different approach here rather than storing references to Server (NSURLProtocol).